README.md

```diff
@@ -87,16 +87,31 @@ Additional methods (not recommended for production):
 - Navigate to **Instances and Subscriptions** -> **Instances** -> **AI Core**
 - Click **View Credentials** and copy the JSON content
 
-**2. Set Environment Variable:**
+**2. Create `.env` file:**
+
+- Create an `.env` file in the root directory of your application
+- Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`
+
+<details>
+<summary>Set an environment variable instead of .env</summary>
+
+**2. Set an Environment Variable:**
```

Comment:
If you want to indicate this as an alternative to step 2, let's make it explicit:

```suggestion
**2. Set an Environment Variable: (alternative)**
```